app/uk/gov/hmrc/tariffclassificationfrontend/controllers/CasesController.scala

```diff
@@ -16,27 +16,22 @@
 
 package uk.gov.hmrc.tariffclassificationfrontend.controllers
 
-import java.time.LocalDate
-
 import javax.inject.{Inject, Singleton}
 import play.api.i18n.{I18nSupport, MessagesApi}
 import play.api.mvc._
-import uk.gov.hmrc.play.bootstrap.controller.FrontendController
 import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
-import uk.gov.hmrc.tariffclassificationfrontend.models.Case
+import uk.gov.hmrc.tariffclassificationfrontend.service.CasesService
 import uk.gov.hmrc.tariffclassificationfrontend.views
 
-import scala.concurrent.Future
-
 @Singleton
-class CasesController @Inject()(val messagesApi: MessagesApi, implicit val appConfig: AppConfig) extends FrontendController with I18nSupport {
-
-  val case1 = Case("1", "Laptops", "Pol's PCs", LocalDate.of(2018,10,1), "OPEN", "BTI", 1)
-  val case2 = Case("2", "Pizza", "Ed's Eatery", LocalDate.of(2018,10,5), "DRAFT", "BTI", 10)
-  val case3 = Case("3", "Pasta", "Stefano's Supermarket", LocalDate.of(2018,10,15), "OPEN", "BTI", 5)
-
-  val cases = Action.async {
-    implicit request => Future.successful(Ok(views.html.cases_list(Seq(case1, case2, case3))))
+class CasesController @Inject()(casesService: CasesService,
+                                val messagesApi: MessagesApi,
+                                implicit val appConfig: AppConfig) extends CommonController with I18nSupport {
+
+  def gateway: Action[AnyContent] = Action.async { implicit request =>
+    casesService.getAllCases.map { cases =>
+      Ok(views.html.gateway_cases(cases))
+    } recover recovery
```

Comment:
I don't think we want to do this `recover recovery` Play already redirects us to an error page without it.